src/libsystemd-network/sd-dhcp6-client.c

```diff
@@ -1331,6 +1343,8 @@ int sd_dhcp6_client_stop(sd_dhcp6_client *client) {
                 return 0;
 
         client_stop(client, SD_DHCP6_CLIENT_EVENT_STOP);
+        if (client->send_release)
+                dhcp6_client_send_message(client);
```

Comment:
Missing error handling. I think it is OK to ignore the failure, but please log about that.